flink-table-store-connector/src/main/java/org/apache/flink/table/store/connector/AbstractTableStoreFactory.java

```diff
@@ -132,9 +133,14 @@ private static void validateFileStoreContinuous(Configuration options) {
     }
 
     static FileStoreTable buildFileStoreTable(DynamicTableFactory.Context context) {
-        FileStoreTable table =
-                FileStoreTableFactory.create(
-                        Configuration.fromMap(context.getCatalogTable().getOptions()));
+        boolean batch =
+                context.getConfiguration().get(ExecutionOptions.RUNTIME_MODE)
+                        == RuntimeExecutionMode.BATCH;
+        Map<String, String> options = new HashMap<>(context.getCatalogTable().getOptions());
+        if (batch) {
+            options.put(CoreOptions.COMMIT_FORCE_COMPACT.key(), String.valueOf(true));
```

Comment:
Does the config option `COMMIT_FORCE_COMPACT` need to explain in the description that the default value is true in BATCH mode?